mis_builder/models/mis_report_instance.py

```diff
@@ -896,7 +911,15 @@ def drilldown(self, arg):
         expr = arg.get("expr")
         account_id = arg.get("account_id")
         if period_id and expr and AEP.has_account_var(expr):
-            period = self.env["mis.report.instance.period"].browse(period_id)
+            period = self.env["mis.report.instance.period"].search_fetch(
```

Comment:
The `browse()` method queries all fields from a table, whereas the `search_fetch()` method queries only the fields that we define. Using `search_fetch()` can help optimize the code.

![image](https://github.com/user-attachments/assets/0a6fa370-7f9a-4933-a2d4-91db6f7b2e72)

Reply:
In terms of performance in terms of query time and resources, the improvement is quite significant.

Talking about improving performance is always a good goal, if you include the reason for this change in PR approach in the PR's description, perhaps it will help the reviewer visualize the reason. Or if it's more ideal in my opinion, you should separate it into a different PR with the IMP label. But overall it's okay for me